lib/routes/index.js

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as vc from '@digitalbazaar/vc';
+import assert from 'assert-plus';
+import {
+  createDiscloseCryptosuite as createEcdsaSd2023DiscloseCryptosuite
+} from '@digitalbazaar/ecdsa-sd-2023-cryptosuite';
+import {DataIntegrityProof} from '@digitalbazaar/data-integrity';
+import express from 'express';
+import {loader} from '../documentLoader.js';
+
+const documentLoader = loader.build();
+
+export const router = express.Router();
+
+router.post('/derive', async (req, res) => {
```

Comment:
It is my understanding that if anything throws in an express handler like this, it will result in an unhandled exception and the server will halt.  If that's correct, it would be good to wrap all of this in a try/catch.

My understanding comes from: https://zellwk.com/blog/async-await-express/

Reply:
Thanks, you're correct. I've updated to use `express-async-errors` package to handle that [d2cd692](https://github.com/digitalbazaar/vc-holder-http/pull/1/commits/d2cd692a418bf2b899e8add1275adc4c4525cb20)

Reply:
based on my quick skim of the readme https://github.com/davidbanham/express-async-errors#readme

This seems to predate async support in express itself.  Also, that package refers to itself as a "hack", I don't think we need a hack at this point in time to solve this issue.

Also, that package hasn't been touched since 2018.

Reply:
@JSAssassin, the `express-async-errors` package doesn't look well maintained.

I recommend putting this at the bottom of the file and then doing `router.post('/derive', asyncHandler(async (req, res) => {...` instead.

```js
function asyncHandler(middleware) {
  return function asyncMiddleware(...args) {
    const result = middleware(...args);
    const next = args[args.length - 1];
    const handleError = (...args) => process.nextTick(() => next(...args));
    return Promise.resolve(result).catch(handleError);
  };
}
```

Reply:
@dlongley @mattcollier package removed [e766024](https://github.com/digitalbazaar/vc-holder-http/pull/1/commits/e766024612f92ec9e423e0a69af7551bbf8ebd62)